README.rst

```diff
@@ -201,69 +93,65 @@ This shows how to create an image suitable for the above exercise.
    # Deploy a seed VM. Should work this time.
    ./dev/seed-deploy.sh
 
+   # Deploying the seed restarts networking interface,
+   # run configure-local-networking.sh again to re-add routes.
+   ./config/src/kayobe-config/configure-local-networking.sh
+
    # FIXME: There is an issue with Bifrost which does not restrict the version
    # of proliantutils it installs.
-   ssh stack@192.168.33.5 sudo docker exec bifrost_deploy pip install proliantutils==2.7.0
-   ssh stack@192.168.33.5 sudo docker exec bifrost_deploy systemctl restart ironic-conductor
+   ssh centos@192.168.33.5 sudo docker exec bifrost_deploy pip install proliantutils==2.7.0
+   ssh centos@192.168.33.5 sudo docker exec bifrost_deploy systemctl restart ironic-conductor
 
    # Clone the Tenks repository.
    git clone https://git.openstack.org/openstack/tenks.git
 
    # Shutdown the seed VM.
    sudo virsh shutdown seed
 
-Now take a snapshot of the VM.
+If required, add any additional SSH public keys to /home/centos/.ssh/authorized_keys
+
+If possible, take a snapshot of the hypervisor instance to speed up this
+process in future.
+You are now ready to deploy from this seed image.
 
-.. _full-deploy:
+A Universe from a Seed
+-----------------------------
 
-Full Deploy
------------
+This shows how to deploy a control plane from a VM image that contains a
+pre-deployed seed VM. Having a snapshot image saves us some time.
```

Comment:
I'd move the closing sentence here to begin the next paragraph, to keep the snapshot-related flow grouped together.